### DISCUSSION TITLE
Toots only to users from Local Instance

### DISCUSSION THREAD

**Discussion by piter104** [mastodon/mastodon#17331] [Q&A]
Hi,
Is it possible / hard to add functionality that would let user send toot to all users from local timeline and not to federated?

**Comment by ClearlyClaire** [comment]
This feature exists in forks (such as [Hometown](https://github.com/hometown-fork/hometown) or [glitch-soc](https://github.com/glitch-soc/mastodon)) but was repeatedly rejected from Mastodon itself
